ci/scripts/github/test_codecov.sh

```diff
@@ -44,10 +44,9 @@ set +e
 # Tests known to be failing
 # Issues:
 # * test_mrc_private - https://github.com/nv-morpheus/MRC/issues/33
-# * nvrpc - https://github.com/nv-morpheus/MRC/issues/34
-ctest --output-on-failure \
-      --exclude-regex "test_mrc_private|nvrpc" \
-      --output-junit ${REPORTS_DIR}/report_ctest.xml
+CODE_COV_RUN=y ctest --output-on-failure \
```

Comment:
This will give inconsistent results when running code coverage locally vs in CI.